assets/src/containers/DashboardAppBar.js

```diff
@@ -64,7 +64,7 @@ function DashboardAppBar (props) {
           <Button>
             <Typography variant='h6' color='inherit' className={classes.grow}>
               <Link
-                to={{ pathname: `/${courseId}` }}
+                to={{ pathname: `/courses/${courseId}` }}
                 className={classes.homeButton}>
                   My Learning Analytics: {courseName}
```

Comment:
Just a thought, what do you think of the idea of making `My Learning Analytics` as a another link and clicking on it goes to `/courses/` page ( which is the list of user courses in Myla) and the `{course_name}` part separate link clicking on it goes to `/courses/{courseId}` ( which is the list if views in a course)

Reply:
That could work, although then we should have two buttons, one for the `My Learning Analytics` and another for the course name. 

Alternatively, we could add an option in the Avatar Modal component that has a way for users to switch between courses.

Reply:
Either way is fine. You can open a separate issue for that fix later. 
LGTM

Reply:
Okay great.